README.md

```diff
@@ -8,3 +8,19 @@ This repository provides a kustomize base to deploy a kafka cluster
 - Zookeeper
 - lag-monitor
 - [mirror-maker 2](mirror-maker-2)
+- [kafka-ui](./kafka-ui)
+
+## Versioning
```

Comment:
I was thinking to put this in a slab document that we can reference from multiple places.
Something like "Versioning common manifests".
As we're using the same for opentelemetry-manifests, https://github.com/utilitywarehouse/cockroachdb-manifests, and I expect more will follow

Reply:
> I was thinking to put this in a slab document that we can reference from multiple places.
Something like "Versioning common manifests".
As we're using the same for opentelemetry-manifests, https://github.com/utilitywarehouse/cockroachdb-manifests, and I expect more will follow

Another solution (I think already discussed): we could just have a single repo for manifests?

Reply:
Indeed, that's also an option. Would make it easier for people to find shared manifests.
Would also be cool to have an easier way to release those. 
Not sure if release-please would be able to handle this format

Reply:
>  Not sure if release-please would be able to handle this format

:thinking: good question, would be convenient if it did. Though if it doesn't then it wouldn't work with individual repos anyway right (i.e. we wouldn't lose anything in this regard by moving to a mono-repo)?

Reply:
I've been using the creation of the changelog from github: https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes. I think this would not work with releasing multiple compoments.
Other than this, no loss I think

Reply:
I guess this PR doesn't hurt to be merged. I see you also updated the cockroach DB one.

Reply:
> I guess this PR doesn't hurt to be merged. I see you also updated the cockroach DB one.

:+1: I think there improvements that could be made:

* Document somewhere more central
* Move to a mono-repo so it's all in one place
* Move to a mono-repo and automate tagging+CHANGELOG generation so we don't have to think about it (this would be ideal)